1-Vanilla/main.js

```diff
@@ -0,0 +1,231 @@
+const list = [
+    {
+      text: "Doing the Dishes",
+      status: "unfinished",
+      isEditMode: false,
+      id: 1,
+    },
+    {
+      text: "Fold Laundry",
+      status: "unfinished",
+      isEditMode: false,
+      id: 2,
+    },
+    {
+      text: "Build Todo App",
+      status: "finished",
+      isEditMode: false,
+      id: 3,
+    },
+    {
+      text: "Get a haircut",
+      status: "unfinished",
+      isEditMode: false,
+      id: 4,
+    },
+    {
+      text: "Call to mom and dad",
+      status: "unfinished",
+      isEditMode: false,
+      id: 5,
+    },
+    {
+      text: "Game day against Elitzur Tel-Aviv",
+      status: "unfinished",
+      isEditMode: false,
+      id: 6,
+    },
+    {
+      text: "Make dinner",
+      status: "unfinished",
+      isEditMode: false,
+      id: 7,
+    },
+    {
+      text: "Eat lunch",
+      status: "finished",
+      isEditMode: false,
+      id: 8,
+    },
+    {
+      text: "Go to Wix offices",
+      status: "unfinished",
+      isEditMode: false,
+      id: 9,
+    },
+    {
+      text: "Watch Deni Avdjia game",
+      status: "unfinished",
+      isEditMode: false,
+      id: 10,
+    },
+    {
+      text: "Go to step 2",
+      status: "unfinished",
+      isEditMode: false,
+      id: 11,
+    },
+    {
+      text: "Infra webinar at 3",
+      status: "unfinished",
+      isEditMode: false,
+      id: 12,
+    },
+    {
+      text: "Pay the bills",
+      status: "unfinished",
+      isEditMode: false,
+      id: 13,
+    },
+    {
+      text: "Contribute code to open source",
+      status: "unfinished",
+      isEditMode: false,
+      id: 14,
+    },
+    {
+      text: "Finish Velo site",
+      status: "unfinished",
+      isEditMode: false,
+      id: 15,
+    },
+  ];
+  
+  const loadToDoList = (list) => {
+    const ul = document.getElementById("todoList");
+    ul.innerHTML = "";
+    list.forEach((item) => {
+      ul.appendChild(createItem(list, item));
+    });
+  };
+  
+  const addTodo = () => {
+    const inputText = document.getElementById("addTodo").value;
+    list.push({
+      text: inputText,
+      status: "unfinished",
+      id: list.length + 1,
+    });
+    loadToDoList(list);
+  };
+  
+  const removeTodo = (list, item) => {
+    list = list.filter(function (ele) {
+      return ele.id != item.id;
+    });
+  
+    loadToDoList(list);
+  };
+  
+  const editTodo = (list, item) => {
+    list = list.map((ele) => {
+      if (ele.id === item.id) {
+        ele.isEditMode = !ele.isEditMode;
+      }
+      return ele;
+    });
+    loadToDoList(list);
+  };
+  
+  const toggleTodo = (item) => {
+    const li = document.getElementById(item.id);
+    const text = li.children[0].children[1];
+    if (item.status === "finished") {
+      text.style["text-decoration"] = "none";
```

Comment:
styling considerations should be exported to css files, use classes!

Reply:
fixed it.